lib/ssl/src/ssl_handshake.erl

```diff
@@ -353,8 +353,9 @@ certify(#certificate{asn1_certificates = ASN1Certs}, CertDbHandle, CertDbRef,
                 path_validation_alert(Reason)
 	end
     catch
-        error:{_,{error, {asn1, Asn1Reason}}} ->
+        error:{_,{error, {asn1, Asn1Reason}}}=OtherReason:ST ->
```

Comment:
So when we get here we no longer decode any certificates here and this error clause should no be dead code!